code/modules/cargo/packs/service.dm

```diff
@@ -83,8 +70,103 @@
+					/obj/item/circuitboard/machine/chem_dispenser/drinks,
+					/obj/item/circuitboard/machine/dish_drive)
+	crate_name = "build a bar crate"
+
+/datum/supply_pack/service/maintgarden
+	name = "Maintenance Garden Crate"
+	desc = "Set up your own tiny paradise with do-it-yourself botany kit. Contains sandstone for dirt plots, pest spray, ammonia, a portable seed generator, basic botanical tools, and some seeds to start off with."
+	cost = 2350
+	contains = list(/obj/item/storage/bag/plants/portaseeder,
+					/obj/item/reagent_containers/spray/pestspray,
+					/obj/item/stack/sheet/mineral/sandstone/twelve,
+					/obj/item/reagent_containers/glass/bucket,
+					/obj/item/reagent_containers/glass/bottle/ammonia,
+					/obj/item/reagent_containers/glass/bottle/ammonia,
+					/obj/item/hatchet,
+					/obj/item/cultivator,
+					/obj/item/plant_analyzer,
+					/obj/item/flashlight,
+					/obj/item/seeds/carrot,
+					/obj/item/seeds/carrot,
+					/obj/item/seeds/tower,
+					/obj/item/seeds/tower,
+					/obj/item/seeds/watermelon,
+					/obj/item/seeds/watermelon,
+					/obj/item/seeds/grass,
+					/obj/item/seeds/grass)
+	crate_name = "maint garden crate"
+	crate_type = /obj/structure/closet/crate/hydroponics
+
+//////////////////////////////////////////////////////////////////////////////
+/////////////////////////////// Janitor //////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/service/advlighting
+	name = "Advanced Lighting crate"
+	desc = "Thanks to advanced lighting tech we here at the Lamp Factory have be able to produce more lamps and lamp items! This crate has three lamps, a box of lights and a state of the art rapid-light-device!"
+	cost = 2750
+	contains = list(/obj/item/construction/rld,
+					/obj/item/flashlight/lamp,
+					/obj/item/flashlight/lamp,
+					/obj/item/flashlight/lamp/green,
+					/obj/item/storage/box/lights/mixed)
+	crate_name = "advanced lighting crate"
+	crate_type = /obj/structure/closet/crate/secure
+
+/datum/supply_pack/service/lightbulbs
+	name = "Replacement Lights" //Subgrouping this with Advanced Lighting Crate, they're both lighting related.
+	desc = "May the light of Aether shine upon this station! Or at least, the light of forty two light tubes and twenty one light bulbs as well as a light replacer."
+	cost = 1200
+	contains = list(/obj/item/storage/box/lights/mixed,
+					/obj/item/storage/box/lights/mixed,
+					/obj/item/storage/box/lights/mixed,
+					/obj/item/lightreplacer)
+	crate_name = "replacement lights"
+
+/datum/supply_pack/service/janitor/advanced
+	name = "Advanced Sanitation Crate"
+	desc = "Contains all the essentials for an advanced spacefaring cleanup crew. This kit includes a trashbag of holding, an advanced mop, a bottle of space cleaner, a floor buffer, and a holosign projector."
+	cost = 5000
+	access = ACCESS_JANITOR
+	contains = list(/obj/item/storage/bag/trash/bluespace,
```

Comment:
Hmm...

Reply:
Is it the access, cost, or the trashbag? I can unlist it from janitor and make it a bit more expensive, or replace the bag with a normal one.

In all honesty, most of the time when I play janitor I don't even need a bluespace bag unless someone is throwing glowsticks everywhere.